src/middlewares/accommodation/accommodationMiddleware.js

```diff
@@ -0,0 +1,27 @@
+import 'dotenv/config';
+import Formidable from 'formidable';
+
+export default async function accommodationMiddleware (req, res, next){
+	
+	if(Object.keys(req.body).length>0){
+		req.inputFiles = {};
+		next();
+	}
+	
+	// parse an uploaded file
```

Comment:
which file do you want to upload?

Reply:
The "uploaded file" mentioned in that comment is a picture of an accommodation facility / it's room.